backend/controllers/googlegroup.controller.js

```diff
@@ -0,0 +1,88 @@
+const GoogleGroup = require('../models/googlegroup.model')
+
+const getAllGoogleGroups = async () => {
+    try {
+        let result = []
+        const allPairs = await GoogleGroup.find()
+        allPairs.map((pair) => result.push(pair))
```

Comment:
redundant code, can just return allPairs. also if you're not going to return things from a map function just use forEach